src/pages/billing/Project.js

```diff
@@ -564,29 +564,36 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const tabToTable = {
     workspaces: h(Fragment, [
-      div({ role: 'table', 'aria-label': `workspaces in billing project ${billingProject.projectName}` }, [
-        h(WorkspaceCardHeaders, {
-          needsStatusColumn: billingAccountsOutOfDate,
-          sort: workspaceSort,
-          onSort: setWorkspaceSort
-        }),
-        div({}, [
-          _.flow(
-            _.orderBy([workspaceSort.field], [workspaceSort.direction]),
-            _.map(workspace => {
-              const isExpanded = expandedWorkspaceName === workspace.name
-              return h(WorkspaceCard, {
-                workspace: { ...workspace, billingAccountDisplayName: billingAccounts[workspace.billingAccount]?.displayName },
-                billingProject,
-                billingAccountStatus: billingAccountsOutOfDate && getBillingAccountStatus(workspace),
-                key: workspace.workspaceId,
-                isExpanded,
-                onExpand: () => setExpandedWorkspaceName(isExpanded ? undefined : workspace.name)
+      !_.isUndefined(workspaces) && _.isEmpty(workspacesInProject) ?
```

Comment:
@yunip20 I changed this so that we don't show the message until we have gotten back the Ajax response for the workspaces and thus no that there truly aren't any. Otherwise you can see the message render and then disappear when the workspaces are loaded.

I also changed it so we show it whenever there are no workspaces in a billing project (not just for owners), and added integration test coverage of that.